web/src/services/i18n/translations/en.yml

```diff
@@ -27,6 +27,15 @@ Remove: Remove
 Add Item: ''
 Please choose an option: ''
 Add text here: ''
+Timeline Settings: 'Timeline Settings'
+'* Start Time': ''
+Start time for the timeline: ''
+'* End Time ': ''
+End time for the timeline: ''
+'* Current Time ': ''
+Current time should be between start and end time: ''
+Please make sure the Current time must between the Start time and End Time.: ''
```

Comment:
Please make sure to copy/paste the english over to the other side too. For the most part it doesn't have an effect, but there have been instances where tests fail because they use the empty string instead of the key